src/redux/email-questions/tests/email-questions.sagas.test.js

```diff
@@ -0,0 +1,19 @@
+import {
+  answerEmailQuestion,
+  deleteEmailQuestions,
+  getAllEmailQuestions,
+  getEmailQuestionById,
+  getPendingEmailQuestionsCount,
+  makeEmailQuestionsSpam
+} from '../email-questions.operations';
+
+describe('email questions sagas test', () => {
+  it(' should not throw error', () => {
+    expect(getAllEmailQuestions).not.toThrow();
+    expect(getEmailQuestionById).not.toThrow();
+    expect(deleteEmailQuestions).not.toThrow();
+    expect(makeEmailQuestionsSpam).not.toThrow();
+    expect(answerEmailQuestion).not.toThrow();
+    expect(getPendingEmailQuestionsCount).not.toThrow();
+  });
+});
```

Comment:
Please add tests for saga, and remove first test because it will cause errors in your future sagas tests for email questions.